{"repository":"https://github.com/shopkeep/shpkpr","dockerfilePath":"Dockerfile","startTime":1673169737247,"endTime":1673169799251,"clone":{"startTime":1673169737247,"stdout":"","stderr":"Cloning into '/tmp/dinghy-analysis/shopkeep/shpkpr'...\n","error":null,"endTime":1673169738014,"commit":"aa380c9d26bee7a21be5e08957534f561116bdf2"},"originalBuild":{"startTime":1673169738089,"endTime":1673169799191,"error":{"code":1,"killed":false,"signal":null,"cmd":"docker build --no-cache --force-rm -t shpkpr:latest -f /tmp/dinghy-analysis/shopkeep/shpkpr/Dockerfile ."},"stdout":"Sending build context to Docker daemon  333.3kB\r\r\nStep 1/6 : FROM python:3.6.1-alpine\n3.6.1-alpine: Pulling from library/python\n90f4dba627d6: Pulling fs layer\n19bc0bb0be9f: Pulling fs layer\ne05eff433916: Pulling fs layer\ne70196200a87: Pulling fs layer\na6d780959950: Pulling fs layer\na6d780959950: Waiting\ne70196200a87: Waiting\n19bc0bb0be9f: Verifying Checksum\n19bc0bb0be9f: Download complete\n90f4dba627d6: Verifying Checksum\n90f4dba627d6: Download complete\ne05eff433916: Verifying Checksum\ne05eff433916: Download complete\n90f4dba627d6: Pull complete\ne70196200a87: Verifying Checksum\ne70196200a87: Download complete\na6d780959950: Verifying Checksum\na6d780959950: Download complete\n19bc0bb0be9f: Pull complete\ne05eff433916: Pull complete\ne70196200a87: Pull complete\na6d780959950: Pull complete\nDigest: sha256:0945574465b917d524ce9b748479a286c2ed3c5a97311ac5950464907d4d8b53\nStatus: Downloaded newer image for python:3.6.1-alpine\n ---> ddd6300d05a3\nStep 2/6 : RUN mkdir -p /usr/src/app\n ---> Running in 6b40f140730a\nRemoving intermediate container 6b40f140730a\n ---> bd09a48df854\nStep 3/6 : WORKDIR /usr/src/app\n ---> Running in 33ff04b8c438\nRemoving intermediate container 33ff04b8c438\n ---> cb2d253d4ec6\nStep 4/6 : COPY . /usr/src/app\n ---> 25f72bd5f8f0\nStep 5/6 : RUN python setup.py install\n ---> Running in bab2e003790f\n\u001b[91mCouldn't find index page for 'pytest-runner' (maybe misspelled?)\n\u001b[0m\u001b[91mNo local packages or working download links found for pytest-runner\n\u001b[0m\u001b[91mTraceback (most recent call last):\n  File \"setup.py\", line 69, in <module>\n    'responses',\n  File \"/usr/local/lib/python3.6/distutils/core.py\", line 108, in setup\n    _setup_distribution = dist = klass(attrs)\n  File \"/usr/local/lib/python3.6/site-packages/setuptools/dist.py\", line 320, in __init__\n    self.fetch_build_eggs(attrs['setup_requires'])\n  File \"/usr/local/lib/python3.6/site-packages/setuptools/dist.py\", line 377, in fetch_build_eggs\n    replace_conflicting=True,\n  File \"/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py\", line 852, in resolve\n    dist = best[req.key] = env.best_match(req, ws, installer)\n  File \"/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py\", line 1124, in best_match\n    return self.obtain(req, installer)\n  File \"/usr/local/lib/python3.6/site-packages/pkg_resources/__init__.py\", line 1136, in obtain\n    return installer(requirement)\n  File \"/usr/local/lib/python3.6/site-packages/setuptools/dist.py\", line 445, in fetch_build_egg\n    return cmd.easy_install(req)\n  File \"/usr/local/lib/python3.6/site-packages/setuptools/command/easy_install.py\", line 667, in easy_install\n    raise DistutilsError(msg)\ndistutils.errors.DistutilsError: Could not find suitable distribution for Requirement.parse('pytest-runner')\n\u001b[0mRemoving intermediate container bab2e003790f\n","stderr":"The command '/bin/sh -c python setup.py install' returned a non-zero code: 1\n"}}